pkg/api/dashboard.go

```diff
@@ -139,7 +139,7 @@ func (hs *HTTPServer) GetDashboard(c *models.ReqContext) response.Response {
 		Url:                    dash.GetUrl(),
 		FolderTitle:            "General",
 		AnnotationsPermissions: annotationPermissions,
-		IsPublic:               dash.IsPublic,
+		IsEnabled:              false,
```

Comment:
@owensmallwood I did a quick grep and I don't think we're using IsEnabled, but if we are. I think we should rename this to IsPublicDashboardEnabled as IsEnabled is opaque at best. CC @pianohacker

Reply:
It is being used as `isPublic` for [PanelHeaderMenuTrigger](https://github.com/grafana/grafana/blob/jalevin/refactor_is_public/public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx#L62) disabling. 
We should change it there or leave it as `isPublic` and calculate its value. True if has 1 or more public dashboards.